server/src/main/java/io/crate/execution/engine/collect/DocValuesAggregates.java

```diff
@@ -160,7 +161,9 @@ public static List<DocValueAggregator> createAggregators(Functions functions,
                         // to the normal aggregation implementation
                         return null;
                     }
-                    aggregationReferences.add(reference);
+                    // Ignore source look-ups because they don't make sense for aggregations
```

Comment:
Can we elaborate why it don't make sense? Like, I think it not only make no real sense but break things!

Reply:
The field does not exist in doc values. I will add it to the comment.